library/src/main/java/com/firebase/ui/auth/core/FirebaseLoginBaseActivity.java

```diff
@@ -0,0 +1,109 @@
+package com.firebase.ui.auth.core;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+
+import com.firebase.client.AuthData;
+import com.firebase.client.Firebase;
+import com.firebase.client.FirebaseError;
+import com.firebase.ui.auth.facebook.FacebookAuthHelper;
+import com.firebase.ui.auth.google.GoogleAuthHelper;
+import com.firebase.ui.auth.password.PasswordAuthHelper;
+import com.firebase.ui.auth.twitter.TwitterAuthHelper;
+
+public abstract class FirebaseLoginBaseActivity extends AppCompatActivity {
```

Comment:
We need docs here, otherwise nobody will know when to use this class, unless they read the README.